packages/@aws-cdk/aws-codepipeline/lib/pipeline.ts

```diff
@@ -328,47 +358,6 @@ export class Pipeline extends cdk.Construct implements cpapi.IPipeline {
     };
   }
 
-  // ignore unused private method (it's actually used in Stage)
-  // @ts-ignore
-  private _generateOutputArtifactName(stage: cpapi.IStage, action: cpapi.Action): string {
```

Comment:
Gotta say (again) that I think we still need this

Reply:
See above comment.